commands/new_function_test.go

```diff
@@ -25,30 +25,43 @@ const ListOptionOutput = `Languages available as templates:
 const LangNotExistsOutput = `(?m:is unavailable or not supported)`
 
 type NewFunctionTest struct {
-	title       string
-	funcName    string
-	funcLang    string
-	expectedMsg string
+	title         string
+	prefix        string
+	funcName      string
+	funcLang      string
+	expectedImage string
```

Comment:
added to test the expected image name (previously it was equal to `funcName`)

Reply:
I'm impressed. Thank you 💯